packages/terra-form-select/src/clinical-lowlight-theme/Frame.module.scss

```diff
@@ -37,6 +37,8 @@
     --terra-form-select-invalid-focus-background-size: 0 0;
     --terra-form-select-invalid-focus-border-color: #fb4c4c;
     --terra-form-select-invalid-focus-box-shadow: 0 0 0 1px #fb4c4c, 0 0 0 1px #fb4c4c;
+    --terra-form-select-single-select-invalid-focus-border-color: #004c76;
+    --terra-form-select-single-select-invalid-focus-box-shadow: 0 0 1px 3px #004c76, 0 0 7px 4px #004c76;
```

Comment:
Were inputs from UX taken regarding these.  On focus the red color disappears which does not seem correct

Reply:
These styles has been removed and added outline around the red colour border when it gets focussed. So that we can see the red colour border as usual with outline.